Exiled.CustomItems/Commands/ListItems.cs

```diff
@@ -0,0 +1,44 @@
+// -----------------------------------------------------------------------
+// <copyright file="ListItems.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.CustomItems.Commands
+{
+    using System;
+
+    using CommandSystem;
+
+    using Exiled.CustomItems.API;
+
+    /// <summary>
+    /// The command to list all installed items.
+    /// </summary>
+    [CommandHandler(typeof(RemoteAdminCommandHandler))]
+    [CommandHandler(typeof(GameConsoleCommandHandler))]
+    public class ListItems : ICommand
+    {
+        /// <inheritdoc/>
+        public string Command { get; } = "citemlist";
+
+        /// <inheritdoc/>
+        public string[] Aliases { get; } = new[] { "clist" };
+
+        /// <inheritdoc/>
+        public string Description { get; } = "Gets a list of all custom items currently installed and enabled on the server.";
+
+        /// <inheritdoc/>
+        public bool Execute(ArraySegment<string> arguments, ICommandSender sender, out string response)
+        {
+            string message = string.Empty;
+            foreach (CustomItem item in CustomItems.Singleton.ItemManagers)
+                message += $"{item.Name}({item.Id})\n";
```

Comment:
Never do that. Get a `StringBuilder` by calling `NorthWoodLib.Pools.StringBuilderPool.Shared.Rent()` and free it by calling `NorthWoodLib.Pools.StringBuilderPool.Shared.Return(StringBuilder)` when you get your string.